course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+# EEB430H1 Theoretical Ecology [24L, 12P]
```

Comment:
Ah ok, do you want me to remove the first line from the syllabus and change the title to "Theoretical Ecology and Reproducible Quantitative Methods using R" instead?